### PR TITLE
Formatting cleanup in Format

### PR DESCRIPTION
**Pull request by sebastianbaginski**
I'm thinking about working on this file soon, so I wanted to get this out of the way
@marcusnaslund review ? this affects only formatting

### CODE REVIEW

File: source/sdk/lang/Format.ooc [hunk 1]
```diff
@@ -22,581 +22,550 @@
...
+				} else {
+					res append("(nil)")
+				}
+		case '%' =>
+			res append('%')
+			mprintCall = false
+		case =>
+			mprintCall = false
+	}
+	if (mprintCall) {
```
Comment (marcusnaslund) [inline source/sdk/lang/Format.ooc]: `this`

File: source/sdk/lang/Format.ooc [hunk 2]
```diff
@@ -22,581 +22,550 @@
...
 }
 
 InvalidTypeException: class extends Exception {
-    init: func (T: Class) { message = "invalid type %s passed to generic function!" format(T name) }
+	init: func (T: Class) { message = "invalid type %s passed to generic function!" format(T name) }
 }
 
-
 /* Text Formatting */
 TF_ALTERNATE := 1 << 0
```
Comment (marcusnaslund) [inline source/sdk/lang/Format.ooc]: Make these nonglobal

File: source/sdk/lang/Format.ooc [hunk 3]
```diff
@@ -22,581 +22,550 @@
...
+			info@ base = 16
+		case 'X' =>
+			info@ base = 16
+		case 'p' =>
+			info@ flags |= TF_ALTERNATE | TF_SMALL
+			info@ base = 16
+		case 'f' =>
+			// reconstruct the original format statement.
+			// TODO let this do the real thing.
+			mprintCall = false
```
Comment (marcusnaslund) [inline source/sdk/lang/Format.ooc]: `this`

File: source/sdk/lang/Format.ooc [hunk 4]
```diff
@@ -22,581 +22,550 @@
...
+						}
+						res append(sval@)
+						sval += 1
+					}
+				} else {
+					res append("(nil)")
+				}
+		case '%' =>
+			res append('%')
+			mprintCall = false
```
Comment (marcusnaslund) [inline source/sdk/lang/Format.ooc]: `this` and on line 253

File: source/sdk/lang/Format.ooc [hunk 5]
```diff
@@ -22,581 +22,550 @@
...
+			info@ base = 16
+		case 'X' =>
+			info@ base = 16
+		case 'p' =>
+			info@ flags |= TF_ALTERNATE | TF_SMALL
+			info@ base = 16
+		case 'f' =>
+			// reconstruct the original format statement.
+			// TODO let this do the real thing.
+			mprintCall = false
```
Comment (marcusnaslund) [inline source/sdk/lang/Format.ooc]: `this` and line 328+330+332

**Comment by marcusnaslund** [comment]
Bah. I started commenting here and there sicne this is a cleanup PR anyway, and it'll be nicer to get those out of the way before you place an editing PR. Do as much or as little as you want. Basically, a bunch of `this` are missing, and a lot of `{}` brackets can be removed. I don't know what your plan is, so if you're happy as it is, just bump it up to `final review`.

**Comment by sebastianbaginski** [comment]
My plan was to prevent unwanted changes when saving this file. Sometimes when editing files from SDK I have changes in diff which I never made manually.
This file is probably gonna change a lot, depends if I can fix leaks in VarArgs, so you can just merge it for now.

**Comment by marcusnaslund** [comment]
Fair enough. Will go upstream in a few minutes.

**Pull request merged**